### DISCUSSION TITLE
Why this config is defined, and never used?

### DISCUSSION THREAD

**Discussion by dpvenkatesh** [vercel/next.js#36936] [Help]
https://github.com/vercel/next.js/blob/c667d2ba7290d5b7b5d44a8e0c0c07a255728685/examples/amp/pages/dog.js#L5

**Comment by icyJoseph** [comment]
I think after Next 9, you can do that export and Next.js picks up on it when building the application, like GSSP, and the others.

https://nextjs.org/docs/advanced-features/codemods#withamp-to-config

> Next.js now has the concept of page-level configuration, so the withAmp higher-order component has been removed for consistency.

Slightly similar to https://nextjs.org/docs/api-routes/api-middlewares#custom-config, but not the same configuration API.

It'd be nice to know all of the page-level configuration options.
